src/MoBi.Core/Serialization/Converter/ModuleConfigurationBuilder.cs

```diff
@@ -1,45 +1,58 @@
 ﻿using System.Linq;
+using DevExpress.Utils.About;
```

Comment:
check if these new ones are really used or not. I don't think so.

If you use Extensions -> ReSharper -> Edit -> Cleanup Code...

This will be removed for you.